test/JDBC/input/datatypes/TestSpatialFun-vu-prepare.txt

```diff
@@ -0,0 +1,123 @@
+
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText('Point(1.0 2.0)', 4326) )
+
+#Positive Test for STGeomFromText with SRID 0
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText('Point(47.65100 -22.34900)', 0) )
+
+#Negative Test for STGeomFromText when SRID is not provided
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText('Point(47.65100 -22.34900)') )
+
+#Negative Test for STGeomFromText when SRID >= 10^6
+#SRID should be between 0 to 999999
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText('Point(47.65100 -22.34900)', 1000000000 ) )
+
+#Negative Test for STGeomFromText with SRID < 0
+#SRID should be between 0 to 999999
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText('Point(47.65100 -22.34900)', -1) )
+
+#Negative Test for STGeomFromText when a coordinate is missing
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText('Point(1.0 )', 4326) )
+
+#Negative Test for STGeomFromText when invalid type is provided
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText('Pnt', 4326) )
+
+#Test for STGeomFromText when null Point is Given -> Returns NBCRow
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STGeomFromText(null, 4326) )
+
+#Negative Test for STGeomFromText when Incorrect cast is provided
+#INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geography::STGeomFromText('Point(47.65100 -22.34900)', 4326) )
+
+#Positive Test for STPointFromText with SRID 4326. Rest are same as STGeomFromText
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STPointFromText('Point(47.65100 -22.34900)', 4326) )
+
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::STPointFromText('Point(1.0 2.0)', 4326) )
+
+
+#Positive Test for Point with SRID 4326
+INSERT INTO SPATIALPOINTGEOM_dttemp (location) VALUES ( geometry::Point(47.65100, -22.34900, 4326) )
+
+
+
+CREATE VIEW point_equality1Temp AS SELECT STEquals( p1.location, p2.location ) AS equality FROM SPATIALPOINTGEOM_dttemp p1 CROSS JOIN SPATIALPOINTGEOM_dttemp p2;
+
+CREATE VIEW isInTemp AS SELECT STContains( p1.location, p2.location ) AS isIN FROM SPATIALPOINTGEOM_dttemp p1 CROSS JOIN SPATIALPOINTGEOM_dttemp p2 ORDER BY p1.location.STX;
+
+CREATE VIEW ValFromGeomTemp AS SELECT location.STArea() FROM SPATIALPOINTGEOM_dttemp ORDER BY location.STX;
+
+CREATE VIEW TextFromGeogTemp AS SELECT  STArea(location) AS Area FROM SPATIALPOINTGEOG_dttemp ORDER BY location.Lat;
+
+CREATE VIEW point_EqualityTemp AS SELECT STEquals( p1.location, p2.location ) AS Equality FROM SPATIALPOINTGEOG_dttemp p1 CROSS JOIN SPATIALPOINTGEOG_dttemp p2 ORDER BY p1.location.Lat;
+
+CREATE VIEW point_inTemp AS SELECT STContains( p1.location, p2.location ) AS isIn FROM SPATIALPOINTGEOG_dttemp p1 CROSS JOIN SPATIALPOINTGEOG_dttemp p2;
```

Comment:
Please use the TSQL syntax here - `a.STContains(b)`, `a.STEquals(b)`, `a.STArea()`

Reply:
resolved